test/specs/listoftallestbuilding.spec.js

```diff
@@ -0,0 +1,37 @@
+import ListOfTallestBuildingPage from "../pageobjects/listoftallbuildings.page";
+
+describe("Wikipedia list of tall buildings page", () => {
+    it("verify the second tallest building and its conversion between meter and feet", () => {
+        browser.url("https://en.wikipedia.org/wiki/List_of_tallest_buildings");
+        ListOfTallestBuildingPage.waitPageToLoad();
+        let secondTallestBuildingNameInfo = ListOfTallestBuildingPage.tallestBuildingsTable.getItemByPosition(
+            2
+        ).value;
+        let expected = "Shanghai Tower";
+        expect(secondTallestBuildingNameInfo.nameLinkText).toEqual(expected);
+        let heightInMeter = Number(secondTallestBuildingNameInfo.heightInMeter);
```

Comment:
This strict number conversion should probably go into your page object since you've already abstracted so many things away into helpers.

Reply:
agree. I will modify accordingly.